exercises/practice/perfect-numbers/.meta/src/example.art

```diff
@@ -0,0 +1,6 @@
+classify: function[n][
+    aliquotSum: sum chop factors n
+    switch aliquotSum < n -> 'deficient ->
+    switch aliquotSum > n -> 'abundant
+                          -> 'perfect
```

Comment:
I've used literals here. Do you think that makes sense?

Reply:
Pretty reasonable. I’ve seen it in other tracks a few times.